lib/bundler/cli/common.rb

```diff
@@ -52,5 +52,14 @@ def self.gem_not_found_message(missing_gem_name, alternatives)
       message += "\nDid you mean #{suggestions}?" if suggestions
       message
     end
+
+    def self.config_gem_version_promoter(definition, opts)
```

Comment:
yeah ... already merged though. I'll try to make a note to modify it along with the outdated work perhaps.